packages/theme/src/__tests__/createThemeValuesProxy.test.ts

```diff
@@ -315,4 +337,50 @@ describe("createThemeValuesProxy tests", () => {
       );
     });
   });
+
+  describe("Typography Values", () => {
+    test("returns typography object when value is top level", () => {
+      const proxied = createThemeValuesProxy(
```

Comment:
Recommend change this createThemeValuesProxy prop support params as object, not by multiple params with index like this

Reason is it will really hard to understand/update which prop doing what when look at this, also it's easy when enhance/scale this function, because we don't need to remember index of each params

Example by below

```suggestion
      const proxied = createThemeValuesProxy(
        value: value,
        style: {},
        fontWeight: 400,
        platform: "android",
        themeColor: "light"
      )
```

Reply:
Makes sense, I'll do that change

Reply:
Done!